### ISSUE TITLE
Contact Form Submission - Suggestion

### ISSUE THREAD

**Issue by maggieliu05** [cpinitiative/usaco-guide#3712]
Someone submitted the contact form!

**URL**: https://usaco.guide/dashboard
**Module**: None
**Topic**: Suggestion
**Message**: 
It would be nice to add a module `Maximum Bipartite Matching` in the gold or platinum division.

**Comment by bqi343** [comment]
sure, maybe platinum or advanced (it hasn't appeared in platinum for a while now). currently it's mentioned in the max flow module: https://usaco.guide/adv/max-flow?lang=cpp

**Comment by SansPapyrus683** [comment]
don’t max flow and max matching reduce to each other?

**Comment by bqi343** [comment]
no, you can't do max flow with max matching. the most naive max matching is simpler to implement

**Comment by SansPapyrus683** [comment]
oh yeah, i meant doesn't max matching reduce to max flow lol
but i feel like this is still too small of a topic to warrant its own module
but we do have like 5 modules just for dp
who am i to say anything

**Comment by bqi343** [comment]
> but i feel like this is still too small of a topic to warrant its own module

maybe. though max matching is allowed by the IOI syllabus, whereas max flow is not. 

Here are some notes about bipartite matching that would be good to include if we add such a module: https://colefranks.github.io/18453/lecturenotes/1-matching-notes.pdf
